Safety/boardfiles/Src/main.cpp

```diff
@@ -164,26 +162,29 @@ int main(void)
   MX_USART1_UART_Init();
   MX_USART2_UART_Init();
   /* USER CODE BEGIN 2 */
-  PPMChannel *ppm = new PPMChannel(MAX_PPM_CHANNELS);
-  PWMChannel *pwm = new PWMChannel();
-  attitudeManager *attMng = new attitudeManager(ppm, pwm);
 
   interchipInit();
-
+  HAL_Delay(1000);
   HAL_NVIC_SetPriority(TIM15_IRQn, 1, 0);
   HAL_NVIC_SetPriority(SPI1_IRQn, 0, 0);
 
   /* USER CODE END 2 */
 
+  /* Call init function for freertos objects (in freertos.c) */
+  MX_FREERTOS_Init();
+
+  /* Start scheduler */
+  osKernelStart();
+
+  /* We should never get here as control is now taken by the scheduler */
+
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
   while (1)
   {
     /* USER CODE END WHILE */
 
   /* USER CODE BEGIN 3 */
-    RSSI_Check(); // Run every time to update CommsFailed value
```

Comment:
Has RSSI_Check() been moved to AM?

Reply:
If so, have we checked if this still works (e.g. if we disconnect RC receiver does this trigger failsafe still?)

Reply:
Have not checked if this works. Is this a precondition to getting this PR merged?